<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Base64</class>
 <widget class="QWidget" name="Base64">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>242</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="inputBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>151</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>68</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>输入</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>30</y>
     <width>68</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>输出</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="outputBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>60</y>
     <width>151</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEncode">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>107</width>
     <height>39</height>
    </rect>
   </property>
   <property name="text">
    <string>加密</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDecode">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>107</width>
     <height>39</height>
    </rect>
   </property>
   <property name="text">
    <string>解密</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
